solutions/bio2/Byway/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { SCodeBlock, SText, STitle } from "components";
+
+
+export const Byway = () => {
+  return (
+    <>
+      <STitle>Abridged Statement</STitle>
+      <SText>
+        You are given an undirected, weighted graph,
+        and you need to find the shortest path from node 
+        $1$ to node $v$, such that you halve the weight of 
+        exactly one edge in the graph.
+      </SText>
+
+      <STitle>Observation</STitle>
+      <SText>
+        Simply halving each edge and running Dijkstra's algorithm
+        for every edge will not work, because this is too slow given the 
+        constraints. However, we can use the property of Dijkstra's
+        algorithm that it finds the shortest path from a source 
+        to all other nodes.
+      </SText>
+      <STitle>Solution</STitle>
+      <SText>
+        Run Dijkstra's algorithm starting at node $1$, and 
+        then again starting at node $v$. Now enumerate all 
+        the edges in the graph, and for each edge, simulate halving 
+        its weight, reaching one end of the edge from node $1$ and the other
+        end from node $v$, and then doing the opposite. This 
+        results in an $O(E+V \log V)$ solution, where $E$ is the number
```

Comment:
doesn't dijkstra run in O(E log V)?

Reply:
i think this would be O(E + E log V) which is just the same O(E log V)

Reply:
there are about 100000 different contradicting answers to this online. The reason I said E+VlogV is because Erekle said this to me like years ago